show-land/sass/layout/_top-nav.scss

```diff
@@ -1,31 +1,26 @@
-.container {
-  .header {
+.main-nav {
+  &__list {
     display: flex;
-    justify-content: space-between;
+    padding-top: 1rem;
+    gap: 1.5rem;
```

Comment:
you can use from our shared standard codes.
1- import shared variables in your main codes.
2- add this code in the start of your current file : 
`@use "sass:list"`
then use this code :
`gap: list.nth($spaces, 6)
`